package.json

```diff
@@ -25,6 +25,7 @@
   },
   "dependencies": {
     "bson": "^1.1.1",
+    "mocha": "^6.2.0",
```

Comment:
why was this added?

Reply:
We wanted to see if that would help Travis with the mocha errors (it seems to be more related to the Node version, however)

Reply:
yeah, the issue is definitely with the Node version so we should remove this.

Reply:
removed